sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -368,6 +369,7 @@ public partial class ServiceBusSessionProcessor : System.IAsyncDisposable
         public virtual bool AutoCompleteMessages { get { throw null; } }
         public virtual string EntityPath { get { throw null; } }
         public virtual string FullyQualifiedNamespace { get { throw null; } }
+        protected virtual Azure.Messaging.ServiceBus.ServiceBusProcessor InnerProcessor { get { throw null; } }
```

Comment:
Unfortunately, it was necessary to expose this for mocking as the Session processor delegates its events to the inner processor.